ios/ReactNativeCameraKit/RealCamera.swift

```diff
@@ -484,8 +485,9 @@ class RealCamera: NSObject, CameraProtocol, AVCaptureMetadataOutputObjectsDelega
             metadataOutput.setMetadataObjectsDelegate(self, queue: DispatchQueue.main)
 
             let availableTypes = self.metadataOutput.availableMetadataObjectTypes
-            let filteredTypes = supportedBarcodeType.filter { type in availableTypes.contains(type) }
-            metadataOutput.metadataObjectTypes = filteredTypes
+            let filteredTypes = supportedBarcodeType.filter { type in availableTypes.contains(type.toAVMetadataObjectType()) }
+
+            metadataOutput.metadataObjectTypes = filteredTypes.map { $0.toAVMetadataObjectType() }
```

Comment:
We are doing `toAVMetadataObjectType` twice for the same objects. Something like that should do the same
```
let filteredTypes = supportedBarcodeType
  .map { $0.toAVMetadataObjectType() }
  .filter { availableTypes.contains($0) }

metadataOutput.metadataObjectTypes = filteredTypes
```